docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -1,73 +1,131 @@
+
+     If the feature isn't available in the self-managed product, select **None**.
+
+     <!-- vale Grafana.GoogleWill = NO -->
+
+   - **SELF-MANAGED VERSION**: Select the version of self-managed product that will include the feature.
+
+     <!-- vale Grafana.GoogleWill = YES -->
+
+     If the feature isn't available in the self-managed product, select **None**.
+
+     If the version isn't available, select **No suitable option** and reach out in the #docs Slack channel so that a maintainer can add a new option.
+
+   - **BODY**: Include an overview of the feature and the problem it solves.
+
+     If you want to view some best practices around what to write here, refer to [Writing guidelines for What’s new](#writing-guidelines-for-whats-new-content).
      Add any images and a link to your public YouTube video here.
      If you need more information on adding an image, refer to [Image, diagram, and screenshot guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/).
 
-   - **DOCUMENTATION URL (OPTIONAL)** - URL to the public documentation for this feature.
-   - **ENABLEMENT VIDEO (OPTIONAL)** - Link to the video used for enablement.
-   - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements. This will only be visible on admin.grafana.com/whats-new/, _not_ the public What's New page.
+   - **DOCUMENTATION URL (OPTIONAL)**: URL to the public documentation for this feature.
+   - **ENABLEMENT VIDEO (OPTIONAL)**: Link to the video used for enablement.
+   - **INTERNAL INFORMATION (OPTIONAL)**: Information for Grafana Labs employees only.
+
+     For example, ProductDNA, Slack channel, FAQ, training documentation, or videos.
+     Used for training and internal announcements.
+     This is only visible on the [internal What's new page](https://admin.grafana.com/whats-new/), _not_ the [public What's New page](https://grafana.com/docs/grafana-cloud/whats-new/).
 
-1. Click **Save**. The entry is now in **Draft** status and a PR is generated in the `grafana/website` repository.
-1. If your entry is ready to publish, proceed to step 4. If your entry requires review, follow these steps:
+1. Click **Save**.
+   The entry is now in **Draft** status and the CMS opens a pull request in the `grafana/website` repository.
+1. If your entry is ready to publish, proceed to step 4.
+   If your entry requires review, follow these steps:
 
    1. In the **Status** drop-down, select **In review.**
 
-      **The Documentation Team does not automatically review these pull requests; teams that create What’s new entries are responsible for determining their own review process.** However, there are two weekly Office Hours meetings offered by the Documentation Team that you’re welcome to attend for guidance and assistance:
+      {{< admonition type="note" >}}
```

Comment:
This looks terrible as an admonition. I vote we just leave it as is:
![image](https://github.com/grafana/writers-toolkit/assets/76437239/f9c7a3da-ce55-43f4-aa0d-67d5ebd95797)

Reply:
That's actually a bug with the admonition shortcode. I've opened https://github.com/grafana/website/pull/18284